airflow/providers/openlineage/plugins/listener.py

```diff
@@ -51,6 +57,15 @@ def __init__(self):
         self.log = logging.getLogger(__name__)
         self.extractor_manager = ExtractorManager()
         self.adapter = OpenLineageAdapter()
+        self._selective_enable = conf.getboolean("openlineage", "selective_enable", fallback=False)
+
+    def _is_lineage_enabled(self, obj) -> bool:
```

Comment:
ATM we have `_is_disabled` at `openlineage/plugins/openlineage.py`, which I understand will enable/disable the actual listener. Would it make sense either to rename this `_is_lineage_enabled` to `_is_selective_lineage_enabled` or to rename the one in `openlineage/plugins/openlineage.py` to `_is_openlineage_listener_enabled` - so it's straightforward to distinguish between them?

Reply:
Renamed.